controllers/shooter.py

```diff
@@ -81,12 +97,13 @@ def searching(self) -> None:
             self.next_state("tracking")
         else:
             # TODO test this
-            # pose: wpilib.geometry.Pose2d = self.chassis.get_pose()
-            # rel: wpilib.geometry.Pose2d = self.TARGET_POSITION.relativeTo(pose)
-            # rel_heading = rel.rotation().radians()
-            # self.turret.scan(rel_heading)
-
-            self.turret.scan(0)  # TODO remove this
+            pose: wpilib.geometry.Pose2d = self.chassis.get_pose()
+            rel: wpilib.geometry.Pose2d = pose - self.TARGET_POSITION
+            rel_heading = (
+                math.atan2(rel.translation().Y(), rel.translation().X())
+                + pose.rotation().radians()
+            )
```

Comment:
It seems like this should be constrained to [-pi,pi], but it might be better to do it like this:

```python
translation = rel.translation()
rel_heading = Rotation2d(translation.y, translation.x) + pose.rotation()